src/plugins/intel_gpu/src/graph/graph_optimizer/add_required_reorders.cpp

```diff
@@ -41,6 +41,8 @@ void add_required_reorders::add_reorder(program& p, program_node* node, program_
         throw std::runtime_error("Internal Error: container index out of range exception.");
     }
     p.add_intermediate(new_reorder_node, *usr, idx);
+
+    usr->get_dependency(idx).set_output_layout(reorder_layout, false);
```

Comment:
* Is it really required fix here? Why is the layout inconsistent?
* If it is required patch, please add a comment.
* If it is required patch, please test with dlbenchmark on previous generation of GPU.

Reply:
As we had discussed, I resolved this issue at reorder_inputs step.
But I think it still has an issue. I will fix this by a separate PR.

Reply:
Removed